docs/tutorial/population_dynamics.rst

```diff
@@ -124,55 +175,84 @@ bins to be plotted as `n_bins`.
-Here, a random dataset with the same size as the original is sampled (with replacement) from the original dataset. This sampled dataset
+As an additional check, we can estimate confidence intervals (CI) for the parameters using bootstrapping.
+Here, a dataset with the same size as the original is randomly sampled (with replacement) from the original dataset. This random sample
 is then fit using the MLE method, just as for the original dataset. The fit results in a new estimate for the model parameters.
-This process is repeated many times, and the distribution of the resulting parameters can be analyzed to estimate certain statistics about the them::
+This process is repeated many times, and the distribution of the resulting parameters can be analyzed to estimate certain statistics about them::
+
+    dwell_2.calculate_bootstrap(iterations=100)
 
-    dwell_2.calculate_bootstrap(iterations=1000)
+    plt.figure()
     dwell_2.bootstrap.plot(alpha=0.05)
+    plt.show()
 
-.. image:: kymo_bind_bootstrap_2.png
+.. image:: figures/population_dynamics/dwell2_bootstrap.png
 
-Here we see the distributions of the bootstrapped parameters. The vertical lines indicate the
-means of the distributions, while the red area indicates the estimated confidence intervals. The `alpha` argument determines
-the CI that is estimated as `100*(1-alpha)` % CI; in this case we're showing the estimate for the 95% CI. The values for the
-lower and upper bounds are the `100*(alpha/2)` and `100*(1-alpha/2)` percentiles of the distributions.
+Here we see the distributions of the bootstrapped parameters, each of which ideally should look like a Normal (Gaussian) distribution.
+The vertical lines indicate the means of the distributions, while the red area indicates the estimated confidence intervals.
+The `alpha` argument determines the CI that is estimated as `100*(1-alpha)` % CI; in this case we're showing the estimate for the 95% CI.
+The values for the lower and upper bounds are the `100*(alpha/2)` and `100*(1-alpha/2)` percentiles of the distributions.
 
-Note, however, that while the means correspond well with the optimized model parameters, the distributions are not symmetric.
-In such a case, the simple method of using percentiles as CI values may not be appropriate. For more advanced analysis,
-the distribution values are directly available through the properties `DwelltimeModel.bootstrap.amplitude_distributions` and
-`DwelltimeModel.bootstrap.lifetime_distributions` which return the data as a `numpy` array with
-shape `[# components, # bootstrap samples]`.
+Clearly the distributions here are not Gaussian. Specifically, the two distributions on the left for the fractional amplitudes
+are split. This analysis strongly indicates that the single exponential model is preferable. We can also look at
+the bootstrap for that model to verify the results are satisfactory::
```

Comment:
I'd mention what makes it indicate that the single exponential is preferable because right now it's kind of up to the reader to guess.

I would argue that it's the fact that there seem to be so many amplitudes estimated at zero (e.g. removing that component entirely).